java-client/barrage/src/main/java/io/deephaven/client/impl/BarrageSessionFactoryConfig.java

```diff
@@ -4,34 +4,65 @@
 package io.deephaven.client.impl;
 
 import io.deephaven.annotations.BuildableStyle;
+import io.deephaven.barrage.flatbuf.BarrageMessageWrapper;
+import io.deephaven.client.grpc.UserAgentUtility;
 import io.grpc.ManagedChannel;
 import org.apache.arrow.memory.BufferAllocator;
 import org.immutables.value.Value.Default;
 import org.immutables.value.Value.Immutable;
 
+import java.util.Collections;
+import java.util.List;
 import java.util.Objects;
 import java.util.concurrent.ScheduledExecutorService;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 @Immutable
 @BuildableStyle
 public abstract class BarrageSessionFactoryConfig {
     private static final SessionConfig SESSION_CONFIG_EMPTY = SessionConfig.builder().build();
 
+    static final List<String> VERSION_PROPERTIES = Collections.unmodifiableList(Stream.concat(
+            FlightSessionFactoryConfig.VERSION_PROPERTIES.stream(),
+            Stream.of(UserAgentUtility.versionProperty("barrage", BarrageMessageWrapper.class)))
+            .collect(Collectors.toList()));
```

Comment:
This project is java11+ right?

```suggestion
    static final List<String> VERSION_PROPERTIES = Stream.concat(
            FlightSessionFactoryConfig.VERSION_PROPERTIES.stream(),
            Stream.of(UserAgentUtility.versionProperty("barrage", BarrageMessageWrapper.class)))
            .collect(Collectors.toUnmodifiableList());
```